[2023-06-23 21:00:16,823] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-23T20:59:54.786855+00:00 [queued]>
[2023-06-23 21:00:16,898] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_zero 2023-06-23T20:59:54.786855+00:00 [queued]>
[2023-06-23 21:00:16,900] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-06-23 21:00:16,902] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2023-06-23 21:00:16,905] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-06-23 21:00:16,966] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_zero> on 2023-06-23T20:59:54.786855+00:00
[2023-06-23 21:00:16,978] {standard_task_runner.py:52} INFO - Started process 1845 to run task
[2023-06-23 21:00:16,998] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_zero', '2023-06-23T20:59:54.786855+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpphcrtpnx', '--error-file', '/tmp/tmpme22pr8l']
[2023-06-23 21:00:17,003] {standard_task_runner.py:77} INFO - Job 9: Subtask task_zero
[2023-06-23 21:00:17,191] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_zero 2023-06-23T20:59:54.786855+00:00 [running]> on host 7f7a4865b07a
[2023-06-23 21:00:17,373] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_zero
AIRFLOW_CTX_EXECUTION_DATE=2023-06-23T20:59:54.786855+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-23T20:59:54.786855+00:00
[2023-06-23 21:00:17,380] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-06-23 21:00:17,383] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pip install beautifulsoup4']
[2023-06-23 21:00:17,408] {subprocess.py:75} INFO - Output:
[2023-06-23 21:00:27,913] {subprocess.py:79} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-06-23 21:00:28,951] {subprocess.py:79} INFO - Requirement already satisfied: beautifulsoup4 in /home/***/.local/lib/python3.6/site-packages (4.12.2)
[2023-06-23 21:00:29,000] {subprocess.py:79} INFO - Requirement already satisfied: soupsieve>1.2 in /home/***/.local/lib/python3.6/site-packages (from beautifulsoup4) (2.3.2.post1)
[2023-06-23 21:01:26,108] {subprocess.py:83} INFO - Command exited with return code 0
[2023-06-23 21:01:26,324] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_zero, execution_date=20230623T205954, start_date=20230623T210016, end_date=20230623T210126
[2023-06-23 21:01:26,556] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-06-23 21:01:26,627] {local_task_job.py:151} INFO - Task exited with return code 0
